demisto_sdk/commands/common/errors.py

```diff
@@ -19,7 +19,7 @@
     'IF100', 'IF106', 'IF115', 'IF116',
     'IN109', 'IN110', 'IN122', 'IN124', 'IN126', 'IN128', 'IN135', 'IN136', 'IN139', 'IN144', 'IN145',
     'MP106',
-    'PA113', 'PA116', 'PA124', 'PA125', 'PA127', 'PA129',
+    'PA113', 'PA116', 'PA124', 'PA125', 'PA127', 'PA129', 'PA130'
```

Comment:
Why did you add it to the ALLOWED_IGNORE_ERRORS?

Reply:
was not sure were to put it, looks like were most of the error and similar errors to mine were. 
if you don't think it's the correct location. plz tell me what you think about it.